interface/themes/style_oemr.css

```diff
@@ -1145,4 +1145,42 @@ height: 300px !important;
     border: solid 1px;
     width: 100%;
     border-collapse: collapse;
-}
\ No newline at end of file
+}
+
+
+.attach-input{
+    text-align: right;
+}
+.full-width{
+    width: 100%;
+}
+.radio-pos{
+    margin-left: 10px;
+}
+.group-row{
+    margin-top: 15px;
+}
+.container-group{
+    margin-top: auto;
+}
+.group-error-msg{
+    color: #ff6643;
+}
+.group-success-msg{
+    color: #4c8029;
+}
+#component-border{
+    margin-bottom: 20px;
+    border: 1px solid #2672ec;
+    width: 100%;
+    min-height: 150px;
+    padding: 10px;
+}
+#main-component ul{
+}
+.float-right{
+    float: right;
+}
+.rtl .float-right{
+    float: left;
+}
```

Comment:
We initially only put the styling in style_light, with the intention of pasting it in to the other styles as well once we finished development. I will do this now.